src/localization/en.json

```diff
@@ -167,7 +167,8 @@
 
   "exhibition": {
     "reception": "Reception",
-    "visitorVariablesManagement": "Visitor variables management",
+    "visitorVariables": "Visitor variables",
+    "resetVisitorVariables": "Reset visitor variables",
```

Comment:
Screen is called manage visitor variables, so shouldn't this be something similar?